### PR TITLE
Add enhanced nuspec elements

### PR DESCRIPTION
**Pull request by alexbestul**
The latest version of GoCdAgent (17.4.0) is awaiting validation with some warnings.

The package generation scripts have already been updated to include a checksum, which was the primary issue with the package, but there are a few other suggestions that don't appear to have been addressed yet.

The suggestions are described on this page in the Chocolatey docs: https://github.com/chocolatey/package-validator/wiki/NuspecEnhancementsMissing

This PR adds  my best guesses at the correct values. The two values I'm least sure of are:
* The docs URL. Should it point to the specific version in the package (ie, https://docs.gocd.io/17.4.0/) or should it point to the latest docs?
* The mailing list URL. Should it point directly to the Google group?

Please let me know if you'd like me to change anything.

### CODE REVIEW

**Comment by ghost** [comment]
[CLA assistant check](https://cla.gocd.io/gocd/gocd-chocolatey?pullRequest=1) <br/>All committers have signed the CLA.

**Comment by arvindsv** [comment]
@varshavaradarajan is the final decision maker on this, but in my view, the mailing list URL should point directly to [the google group](https://groups.google.com/forum/#!forum/go-cd) and the docs URL should be [/current](https://docs.gocd.io/current/).

File: gocd-agent/agent.nuspec
Comment (varshavaradarajan) [inline gocd-agent/agent.nuspec]: Can you please change this to be `https://github.com/gocd/gocd-chocolatey/issues`

File: gocd-agent/agent.nuspec
Comment (varshavaradarajan) [inline gocd-agent/agent.nuspec]: https://groups.google.com/forum/#!forum/go-cd

File: gocd-server/server.nuspec
Comment (varshavaradarajan) [inline gocd-server/server.nuspec]: Same as above.

File: gocd-server/server.nuspec
Comment (varshavaradarajan) [inline gocd-server/server.nuspec]: Same as what's given above

**Review by varshavaradarajan — CHANGES_REQUESTED** [review]
Hi, 

Thank you for sending this PR. I have a couple of small comments. Will merge it post those changes are made. Thanks!

**Comment by alexbestul** [comment]
Varsha,

Most other packages I've looked at use the bugTrackerUrl to link to the software's bug tracker, not the Chocolatey package's bug tracker.  Many of these packages are maintained by the Chocolatey team, so I think they are a good example to follow.

Here are a few examples.  Look for the 'Software Issues' link in the left column:
* https://chocolatey.org/packages/nodejs
* https://chocolatey.org/packages/Firefox
* https://www.chocolatey.org/packages/notepadplusplus.install

Does that change your opinion at all?

Once you confirm, I'll update the PR.

Thanks for the feedback,
-Alex

**Comment by varshavaradarajan** [comment]
Alright. But it depends whether a person has issues with the software or with installing the software with chocolatey. Keep it as https://github.com/gocd/gocd/issues now and based on what the issue is, we can move it to the appropriate place if required. Thanks.

**Comment by alexbestul** [comment]
I have updated the mailingListUrl, as you requested, and left the bugTrackerUrl pointing to the main GoCD issues page.

Please let me know if you'd like the revisions squashed into a single commit, or if there's anything else you'd like me to do.

**Pull request merged**
